OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -268,6 +268,7 @@ public interface ILoadsPalettes { void LoadPalettes(WorldRenderer wr); }
 	public interface ILoadsPlayerPalettes { void LoadPlayerPalettes(WorldRenderer wr, string playerName, HSLColor playerColor, bool replaceExisting); }
 	public interface IPaletteModifier { void AdjustPalette(IReadOnlyDictionary<string, MutablePalette> b); }
 	public interface IPips { IEnumerable<PipType> GetPips(Actor self); }
+	public interface ITextPip { Tuple<PipType, string> GetPip(Actor self); }
```

Comment:
Please create a type containing the returned data instead of using Tuple in a public interface.